packages/common/src/ui/components/buttons/standard/ButtonWithIcon.tsx

```diff
@@ -41,7 +41,7 @@ export const ButtonWithIcon: React.FC<ButtonWithIconProps> = ({
   const text = shrink ? null : t(labelKey, labelProps);
 
   return (
-    <Tooltip title={t(labelKey)}>
+    <Tooltip disableHoverListener={!shrink} title={t(labelKey, labelProps)}>
```

Comment:
Disables the tooltip on icon buttons when not shrunk and makes the tooltip properly translated